### DISCUSSION TITLE
Adding a domain Bug.

### DISCUSSION THREAD

**Discussion by imadaitelarabi** [vercel/community#6335] [Account Issue] [answered]
### Question

My friend had  a domain name in an old vercel account, and  he decide to delete that domain from that old account, so i can add it in my new vercel account, when i tried to add the domain in my account the platform shows me that the domain is belong to another Vercel account

The domain is completly deleted from my friend account, and when we tried to add it in the  old vercel account its says that there's another vercel account that use that domain, and my friend own that domain in namecheap

Please help:
![image](https://github.com/vercel/community/assets/95383805/00596c97-3364-4622-99e4-5d57df97b434)

**Comment by amyegan** [comment, marked as answer]
Hey @imadaitelarabi. If you go to [vercel.com/account/domains](https://vercel.com/account/domains), do you see the domain listed there?  In that case you may need to move the domain to your new hobby team. You can read more about this account structure update in https://github.com/vercel/community/discussions/6298

**Reply by imadaitelarabi** [reply]
Thank you for your help, its done
